<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckpointForm</class>
 <widget class="QDialog" name="CheckpointForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>79</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>NORTH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>WEST</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SOUTH</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EAST</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please select a direction for the checkpoint</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OkButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>80</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>111</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Checkpoint num</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>42</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
